rwlib/source/job/WorkContext.hpp

```diff
@@ -69,13 +63,13 @@ class GameWorld;
  */
 class WorkContext
 {
-	std::unique_ptr<LoadWorker> _worker;
+	std::mutex _inMutex;
+	std::mutex _outMutex;
 
 	std::queue<WorkJob*> _workQueue;
 	std::queue<WorkJob*> _completeQueue;
 
-	std::mutex _inMutex;
-	std::mutex _outMutex;
+	std::unique_ptr<LoadWorker> _worker;
```

Comment:
Might be worth adding a comment somewhere that the order is important here.
I guess the average user wouldn't think of it.